src/main/java/step3/ResultView.java

```diff
@@ -0,0 +1,17 @@
+package step3;
+
+import java.util.List;
+
+public class ResultView {
+
+    public ResultView() {
+        System.out.println("실행 결과");
+    }
+    
+    public void printOutResult(List<Car> carList) {
+        for (Car car : carList) {
+            System.out.println(StringUtil.printToHyphen(car.getCurrentPosition()));
+        }
+        System.out.println("");
```

Comment:
```suggestion
        System.out.println();
```
empty string 이 필요없어 보여요 😀